central/node/datastore/internal/store/common/parts.go

```diff
@@ -11,23 +11,21 @@ var (
 
 // NodeParts represents the pieces of data in an node.
 type NodeParts struct {
-	node *storage.Node
+	Node *storage.Node
 
-	children []*ComponentParts
-	// nodeCVEEdges stores CVE ID to *storage.NodeCVEEdge object mappings
-	nodeCVEEdges map[string]*storage.NodeCVEEdge
+	Children []*ComponentParts
 }
 
 // ComponentParts represents the pieces of data in a component.
 type ComponentParts struct {
-	edge      *storage.NodeComponentEdge
-	component *storage.ImageComponent
+	Edge      *storage.NodeComponentEdge
```

Comment:
how come we are exposing these?

Reply:
The whole split-merge code is moved to common place and will be used by dackbox and postgres.